asclepius/asclepius/dinto.py

```diff
@@ -57,14 +57,25 @@ def sparqled(*args, **kwargs):
     return sparqled
 
 
+@lru_cache()
 @sparql
-def all_drugs():
+def drugs(labels=None):
+
+    if labels is not None:
+        if not isinstance(labels, frozenset):
```

Comment:
Not sure we need to catch this, we should accept any hashable, if it's not hashable then the lru_cache will throw anyway